src/components/AddPlaidBankAccount/index.js

```diff
@@ -106,8 +50,17 @@ class AddPlaidBankAccount extends React.Component {
     }
 
     componentDidMount() {
+        // If we're coming from Plaid OAuth flow then we need to reuse the existing plaidLinkToken
+        // Otherwise, clear the existing token and fetch a new one
+        if (this.props.receivedRedirectURI || this.props.plaidLinkToken) {
+            return;
+        }
+
+        const redirectURI = this.props.isBusinessBankAccount
+            ? `http://localhost:8080/partners/plaid/oauth_web/${CONST.BANK_ACCOUNT.BUSINESS}`
+            : `http://localhost:8080/partners/plaid/oauth_web/${CONST.BANK_ACCOUNT.PERSONAL}`;
```

Comment:
I guess these urls are for testing and will be updated later?

Reply:
Yes, going to adjust these to be staging/prod

Reply:
Also you can probably only change the last bit of the URL depending on the `CONST` rather than the whole URL.

Reply:
Also you can probably only change the last bit of the URL depending on the `CONST` rather than the whole URL.